pkg/function/data_test.go

```diff
@@ -614,3 +634,53 @@ func (s *DataSuite) TestDescribeBackupsRepoNotAvailable(c *C) {
 	out2 := runAction(c, bp2, "describeBackups", tp)
 	c.Assert(out2[DescribeBackupsRepoDoesNotExist].(string), Equals, "true")
 }
+
+func (s *DataSuite) TestCheckRepository(c *C) {
+	tp, _ := s.getTemplateParamsAndPVCName(c, 1)
+
+	// Test backup
+	bp := *newBackupDataBlueprint()
+	out := runAction(c, bp, "backup", tp)
+	c.Assert(out[BackupDataOutputBackupID].(string), Not(Equals), "")
+	c.Assert(out[BackupDataOutputBackupTag].(string), Not(Equals), "")
+
+	// Test CheckRepository
+	bp2 := *newCheckRepositoryBlueprint()
+	out2 := runAction(c, bp2, "checkRepository", tp)
+	c.Assert(out2[CheckRepositoryPasswordIncorrect].(string), Not(Equals), "")
```

Comment:
what should this output be? is it possible to use `Equals` instead of `Not(Equals)`

Reply:
👍

Reply:
Output needs to be `false` is everything is perfect. Updating test